.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,21 @@
-##### SUMMARY
-<!--- Describe the change below, including rationale and design decisions -->
 
+<!--- Describe the change above, including rationale and design decisions -->
 <!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->
 
-##### ISSUE TYPE
+### Type of pull request
 <!--- Pick one below and delete the rest -->
-- Bugfix Pull Request
-- Docs Pull Request
-- Feature Pull Request
-- New Module Pull Request
+- Bugfix
+- Documentation
+- Feature
+- New Module
 
-##### COMPONENT NAME
```

Comment:
I'm not sure whether it's needed. Is there a link with the decision?

Reply:
@gundalow said it was unused in Pull Requests.

Reply:
ah, ok